deps.edn

```diff
@@ -5,8 +5,8 @@
  {com.timeplus/proton-jdbc {:mvn/version "0.6.0"}
```

Comment:
Hi @jovezhong 

There are several differences between `proton-java-driver` and `timeplus-native-jdbc` which add up to make this is a non-trivial feature that deserves its own issue and PR.

| Attribute | `proton-java-driver` | `timeplus-native-jdbc` |
|--------------|-----------------|-----------------------|
| Port  | `8123` | `8463` |
| Driver URL |  `jdbc:proton://localhost:8123` | `jdbc:timeplus://localhost:8463` | 
| Driver Class Name | `com.timeplus.proton.jdbc.ProtonDriver` | `com.timeplus.jdbc.TimeplusDriver` |
| `java.sql.Connection` Impl | [`com.timeplus.proton.jdbc.ProtonConnection`](https://github.com/timeplus-io/proton-java-driver/blob/develop/proton-jdbc/src/main/java/com/timeplus/proton/jdbc/ProtonConnection.java) |  [`com.timeplus.jdbc.wrapper.SQLConnection.java`](https://github.com/timeplus-io/timeplus-native-jdbc/blob/main/timeplus-native-jdbc/src/main/java/com/timeplus/jdbc/wrapper/SQLConnection.java) |

Each of these attributes (port, driver URL, driver class name and `java.sql.Connection` implementation) for Proton are **hard-coded** in the *.clj files of `metabase-proton-driver` so this is not a simple search and replace.

It is a feature that will need to be properly tested to iron out any differences between the two drivers since their `java.sql.Connection` implementations expose several methods, many of which do not overlap.